doc/user/radial_and_axial_expansion.rst

```diff
@@ -60,27 +57,194 @@ length, :math:`L_0`,
+   :name: thermal_exp_comp_properties
+
+   * - Property
+     - Example
+   * - Material
+     - Steel
+   * - Radius
+     - 0.25 cm
+   * - Height
+     - 5.0 cm
+   * - Reference Temperature
+     - 20°C
+   * - Density
+     - 1.0 g/cc
+   * - Mean Coefficient Thermal Expansion
+     - 2 x 10-6 1/°C
+
+The figure below illustrates the thermal expansion phenomena in both the radial and axial
+directions.
+
+.. figure:: /.static/axial_expansion_simple.png
+   :name: therm_exp_illustration
+   :scale: 50%
+
+    Illustration of radial (isotropic) and axial thermal expansion for a cylinder in ARMI.
+
+Thermal expansion calculations are performed for each component in the ARMI reactor data model as
+component temperatures change. Since components are constrained within blocks, the height of
+components are determined by the height of their parent block. The equations below for
+``hot_radius`` through ``hot_density`` illustrate how the radius, height, volume, density, and mass
+are updated for a component during thermal expansion, respectively.
+
+.. list-table:: Example Calculation of Radial and Axial Thermal Expansion for a Cylindrical Component
+   :widths: 33 33 33
+   :header-rows: 1
+
+   * - Component Temperature
+     - 20°C
+     - 1000°C
+   * - Radius
+     - 0.25 cm
+     - 0.251 cm
+   * - Height
+     - 5.0 cm
+     - 5.01 cm
+   * - Volume
+     - 0.982 cc
+     - 0.988 cc
+   * - Density
+     - 1.0 g/cc
+     - 0.994 g/cc
+   * - Mass
+     - 0.982 g
+     - 0.982 g
+
+.. math::
+   :name: hot_radius
+
+    r(T_h) = 0.25 \left(1 + \left(2\times 10^{-6}(1000 − 20)\right)\right) = 0.251 cm
+
+.. math::
+   :name: hot_height
+
+    h(T_h) = 5.0 \left(1 + \left((2\times 10^{-6}(1000 − 20)\right)\right) = 5.01 cm
+
+.. math::
+   :name: hot_volume
+   
+    V(T_h) = \pi (0.251)^2 5.01 = 0.988 cm^3
+
+.. math::
+   :name: hot_density
+
+    \rho(T_h) = \frac{1.0}{\left(1 + 2\times 10^{-6}(1000 − 20)\right)} = 0.994 \frac{g}{cc}
+
+.. math::
+   :name: hot_mass
+
+    m(T_h) = 0.994 \times 0.988 = 0.982 g
+
+Radial thermal expansion occurs for each component in a given block. Mechanical contact between
+components is not accounted for, meaning that the radial expansion of one Component is independent
+from the radial expansion of the others. Solid components may be linked to gas/fluid components
+(i.e., sodium bond, helium) and the gas/fluid area is allowed to expand and contract with changes in
+Component temperatures. 
+
+When two or more components exist within the Block, the overall height change of the Block is driven
+by an axial expansion "target Component" (e.g., fuel). This target is selected (either automatically
+based on component flags or manually in the blueprints) to ensure that a specific component has its
+mass conserved post-expansion. Note, the remaining components also have their mass conserved on the
+assembly-level; however intra-block mass conservation is lost as there is a redistribution of mass
```

Comment:
I believe that this sentence about mass conservation on the assembly level is not true. The mass of non-target components will change in the top-most non-dummy block, because the non-target components will be expanded to the same height as the target component.

Reply:
I think this depends on what the top most block is. If the top most (non-dummy) block is a single solid component, you'll conserve just fine. But if there are multiple solid components, you are right.

In reality, in order to conserve mass, the blueprints need to be set up \*just\* right. Thus the reason for the `detailedAxialExpansion` [blueprints](https://github.com/terrapower/armi/tree/main/armi/tests/detailedAxialExpansion). The axial expansion functionality definitely, without a doubt, will not conserve mass for all set ups.